app/static/js/userProfile.js

```diff
@@ -64,6 +65,24 @@ $(document).ready(function(){
     $("#noteModal").modal("toggle");
   });
 
+  $("#addVisibility").click(function() { 
+    var bonnerChecked = $("input[name='bonner']:checked").val()
+    console.log(bonnerChecked)
+  if (bonnerChecked == 'on') {
+    $("#noteDropdown").addClass('d-none')
+    $("#noteDrop").removeClass('d-none')
+    $("#noteD").hide()
+    
+  } else {
+    $("#noteD").show()
+    $("#noteDropdown").removeClass('d-none')
+    $("#noteDrop").addClass('d-none')    
```

Comment:
Let's be consistent with `hide()` and `show()` everywhere instead of `removeClass` and `addClass`